Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN go mod download
 COPY main.go main.go
 COPY api api
 COPY controllers controllers
+COPY internal internal
 COPY pkg pkg
```

Comment:
We can probably remove `pkg` from the container image.

Also we should probably move all the files in `pkg` to another location.

Reply:
i was going to leave to @erusso7 , since he is going to introduce client pkg, which will naturally go into internal and will allow smooth removal if test